compiler/resolution/implementForallIntents.cpp

```diff
@@ -717,12 +717,15 @@ std::map<ForallStmt*, std::set<Symbol*>> refMaybeConstForallPairs;
 // It is done on an already-existing, explicit shadow variable
 // or before an implicit shadow variable is to be created.
 //
-static void resolveShadowVarTypeIntent(ForallStmt* fs,
+// Returns true if the resolved shadow var intent was an implicit ref-maybe-const
+//
```

Comment:
drop?

Reply:
I meant the empty comment line initially, but if you change the return back to `void` as I requested, maybe the comment is moot altogether.